components/uniset/src/uniset.rs

```diff
@@ -0,0 +1,307 @@
+use std::{
+    char::{from_u32, MAX},
+    convert::TryFrom,
+    ops::RangeBounds,
+    slice::Iter,
+};
+
```

Comment:
FYI: For benchmarks: https://github.com/unicode-org/icu4x/pull/141/files#diff-d178ef05d0af2f09309e480e84f104eb

@zbraniecki can share some more links.

Reply:
implemented in latest commit